### PR TITLE
refactor: Descriptions on Well-Known Annotations/Labels/Taints

### PR DESCRIPTION
**Pull request by jai**
Fixes https://github.com/kubernetes/website/issues/32990

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This document serves both as a reference to the values and as a coordination poi
 
 Example: `app.kubernetes.io/component=database`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:22]: Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).

```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/created-by=controller-manager`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:32]: Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -39,17 +39,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/instance=mysql-abcxzy`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:42]: Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -39,17 +39,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
 
-A unique name identifying the instance of an application.
+A unique name identifying the instance of an application. If you want to declare what application this is part of but not in a unique way, use [app.kubernetes.io/name](##app-kubernetes-io-name).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:44]: Use [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language), Markdown nit

```suggestion
A unique name identifying the instance of an application. To assign a non-unique name, use [app.kubernetes.io/name](#app-kubernetes-io-name).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 5]
```diff
@@ -39,17 +39,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
...
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
 ### app.kubernetes.io/managed-by
 
 Example: `app.kubernetes.io/managed-by=helm`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:52]: Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 6]
```diff
@@ -59,17 +59,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/name=mysql`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
 
-The name of the application.
+The name of the application. Note that this label is typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/).
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:64]: Use simple and direct language, and use consistent text and formatting with other examples of this language.

```suggestion
The name of the application (typically used on [workload resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/).
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 7]
```diff
@@ -79,9 +79,14 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/version="5.7.21"`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
+
+The current version of the application. The values for this field are up to you.
```
Comment (zacharysarah) [inline content/en/docs/reference/labels-annotations-taints/_index.md:84]: Use simple and direct language.

```suggestion
The current version of the application.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@jai Thanks for your patience, and thanks for adding clarity to the page.

I started doing individual suggestions, but gave up about two-thirds of the way through. The feedback is identical to my other suggestions on this PR: de-capitalize all instances of "Workload Resources" and use relative paths for all internal site links.

Otherwise LGTM

**Comment by jai** [comment]
@zacharysarah thanks for taking the time to comment, I'll get on this!

**Comment by reylejano** [comment]
I'm closing this PR as this PR has not addressed comments/suggestions
Please feel free to reopen (with `/reopen`) when the author is ready to address the comments/suggestions
/close

**Pull request closed without merging**